### PR TITLE
Add Make target to create Jenkins instance

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

**Comment by akram** [comment]
/assign @akram 

@ldimaggi we already have an example of jenkins instance under https://github.com/redhat-developer/openshift-jenkins-operator/blob/master/deploy/crds/jenkins_v1alpha1_jenkins_cr.yaml#L1 .
Maybe you can reference this file instead: deploy/crds/jenkins_v1alpha1_jenkins_cr.yaml .
This file is generated by the operator-sdk when we scaffold the project.

File: jenkins-instance.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+--
```
Comment (waveywaves) [inline jenkins-instance.yaml]: We already have `deploy/crds/jenkins_v1alpha1_jenkins_cr.yaml` for `example-jenkins` can we continue using that itself  ?

File: Makefile [hunk 2]
```diff
@@ -324,13 +324,18 @@ deploy-clean:
...
 
 .PHONY: install-operator-source
 ## Install the Jenkins Operator
 install-operator-source:
 	$(Q)kubectl apply -f jenkins-operator-source.yaml
 
+.PHONY: create-jenkins-instance
+## Create the Jenkins Instance
+create-jenkins-instance:
+	$(Q)kubectl apply -f jenkins-instance.yaml
```
Comment (waveywaves) [inline Makefile]: Let's continue using `deploy/crds/jenkins_v1alpha1_jenkins_cr.yaml ` ?

Comment (ldimaggi) [reply Makefile]: Done! Thx for the suggestion!

**Pull request merged**